### ISSUE TITLE
🐛 Installing pyglotaran-extras within a conda environment

### ISSUE THREAD

**Issue by avrabyt** [glotaran/pyglotaran-extras#105]
### Version information

- pyglotaran version: `glotaran, version 0.6.0`
- pyglotaran-extras version: Installation error - `conda install pyglotaran-extras `
- Python version: `Python 3.9.0`
- Operating System: `macOS Monterey 12.5.1`

### Describe the bug

Installation error 

```console
Collecting package metadata (current_repodata.json): done
Solving environment: failed with initial frozen solve. Retrying with flexible solve.
Collecting package metadata (repodata.json): done
Solving environment: failed with initial frozen solve. Retrying with flexible solve.

PackagesNotFoundError: The following packages are not available from current channels:

  - pyglotaran-extras

Current channels:

  - https://repo.anaconda.com/pkgs/main/osx-64
  - https://repo.anaconda.com/pkgs/main/noarch
  - https://repo.anaconda.com/pkgs/r/osx-64
  - https://repo.anaconda.com/pkgs/r/noarch

To search for alternate channels that may provide the conda package you're
looking for, navigate to

    https://anaconda.org

and use the search bar at the top of the page.
```

### To Reproduce

1. Run '...'

```console

# Create virtual environment
conda create -n pyglotaran python==3.9
# Activate 
conda activate pyglotaran
# Install pyglotaran
conda install -c conda-forge pyglotaran
# Install pyglotaran-extras
conda install pyglotaran-extras 
```

2. See error. 

```
Collecting package metadata (current_repodata.json): done
Solving environment: failed with initial frozen solve. Retrying with flexible solve.
Collecting package metadata (repodata.json): done
Solving environment: failed with initial frozen solve. Retrying with flexible solve.

PackagesNotFoundError: The following packages are not available from current channels:

  - pyglotaran-extras

Current channels:

  - https://repo.anaconda.com/pkgs/main/osx-64
  - https://repo.anaconda.com/pkgs/main/noarch
  - https://repo.anaconda.com/pkgs/r/osx-64
  - https://repo.anaconda.com/pkgs/r/noarch

To search for alternate channels that may provide the conda package you're
looking for, navigate to

    https://anaconda.org

and use the search bar at the top of the page.

```

**Comment by avrabyt** [comment]
Update : With pip works fine 🚀 

`pip install pyglotaran-extras`

**Comment by s-weigand** [comment]
@avrabyt Thanks for the issue, the problem is `pyglotaran-extras` ofc is also only on the `conda-forge` channel and we forgot to add the `-c conda-forge`. The readme will be fixed in #108
